tests/test_declarations/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -524,13 +508,10 @@ describe.each\`
 
 // an example to demo multiline quasi
 describe.each\`
-  a    | b    | expected
-  \${11111111111} | \${a()
-  .b()
-  .c()
-  .d()} | \${2}
-  \${1} | \${2} | \${3}
-  \${2} | \${1} | \${3}
+  a              | b                  | expected
+  \${11111111111} | \${a().b().c().d()} | \${2}
```

Comment:
This change causes the test to fail to demonstrate the case it’s supposed to demonstrate. Can you alter the input here so it still breaks?